infra/k8s/delete.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+set -o errexit
+set -o pipefail
+
+set -e
+
+err_report() {
+    echo "Error on line $1"
+}
+
+trap 'err_report $LINENO' ERR
+
+vpcId=`aws ec2 describe-vpcs --filters Name=tag:Name,Values=$NAME | jq ".Vpcs | .[] | .VpcId" | tr -d "\""`
```

Comment:
@coryschwartz any ideas/suggestions on how to fetch `vpcId` `securityGroupId` and `subnetId` without introducing a dependency on `jq` ?

Reply:
Not a whole lot better, really:

```
vpcId=$(aws ec2 describe-vpcs --output text | awk '/VPCS/ {print $8 }')
```

Reply:
I wasn't aware of this `--output` flag, thanks a lot, I think I'd rather rely on `awk` than on `jq` :)